website/docs/feature-flag-tutorials/use-cases/user-management-access-controls.md

```diff
@@ -0,0 +1,172 @@
+
+This automation creates an access control system that adapts in real-time to organizational changes. Within minutes of joining, a new team member can be granted precisely the right level of access, while departing employees are immediately locked out of sensitive systems.
+
+## Configure role-based access controls
+
+[Role-based access control](/reference/rbac) makes feature flag management a strategic governance tool. Consider a scenario in which a junior developer should never be able to modify critical feature flags in the authentication system within a banking platform. RBAC makes this granular control seamless. We recommend carefully delegating administrative privileges to users based on the needs of their roles within projects and the organization.
+
+In the Unleash Admin UI, go to **Admin > Roles** to view, create, and manage user roles.
+
+![Manage all user roles in the Unleash Admin UI.](/img/use-case-user-mgmt-root-roles.png)
+
+We have 5 [predefined roles](/reference/rbac#predefined-roles) within our RBAC framework at Unleash.
+
+_Root roles_:
+
+1. Admin
+2. Editor
+3. Viewer
+
+_Project roles_:
+
+1. Owner
+2. Member
+
+Assign users with root roles to configure entire systems. These users can perform any operation within the Unleash platform and change permissions for other users when they belong to a specific user group.
+
+Assign other users to project roles to have domain-specific control within a specific project maintained in Unleash.
+
+Project permissions are separated from root permissions to make it even more targeted regarding what permissions someone can and cannot have for each piece of Unleash. Assign developers with creation and modification rights and viewers who can observe but not change.
+
+For more fine-tuned access controls, create [custom root roles](/how-to/how-to-create-and-assign-custom-root-roles) and [custom project roles](/how-to/how-to-create-and-assign-custom-project-roles), where you can define the privileges and limitations beyond the predefined roles we have built into Unleash.
+
+### Extend RBAC with a change management workflow
+
+While RBAC allows you to administer Unleash safely, you might need approval processes when changing feature flags or their configuration. When multiple teams are working on complex systems, [change requests](/reference/change-requests) provide a systematic approach to:
+
+-   **Comprehensive review**: Every proposed feature flag modification goes through a review, reducing the likelihood of unintended consequences.
+-   **Audit trail**: Every change is documented, timestamped, and attributed to specific team members, creating a permanent record of system modifications.
+-   **Four-eyes approval workflows**: Multi-stage approval processes ensure critical changes are thoroughly reviewed and approved by at least 2 other people before implementation.
```

Comment:
Is it really that they are "approved by at least 2 **other** people before implementation."?  That would be 6 eyes. ;). I think, but am not sure, that you can have the requestor and a single approver.  Please confirm.

Reply:
Definitions online say "at least 2 people need to approve" and yes, it is vague as to whether or not the requestor can be considered one of the "pairs of eyes".

However, looking a second time (post-holiday freshness :) ), I see that the [Open Risk Manual](https://www.openriskmanual.org/wiki/Four_Eyes_Principle) implies that 1 person needs to review, making it a total of 2 people involved.